pages/en/contrib/style-guide.md

```diff
@@ -449,6 +449,44 @@ async function getDodo() {
 }
 ```
 
+### Always await the last promise in a function
+
+If the last line of a function handles a promise, always await that promise, regardless whether the result will be returned or not. (This makes it clear whether we did or did not intend to return the value, and dismisses any idea that we are still linking up the promise chain in pre-await style.)
+
```

Comment:
Is it better to rephrase it as `This makes it clear whether we intend to return the value or not, and dismisses any idea that we are still linking up the promise chain in pre-await style.`?

Reply:
Thanks for your clearer rewording. Updated.